ape_hardhat/providers.py

```diff
@@ -522,7 +522,7 @@ def build_command(self) -> List[str]:
 
         return cmd
 
-    def reset_fork(self):
+    def reset_fork(self, new_block_number: int):
```

Comment:
```suggestion
    def reset_fork(self, block_number: int):
```

I think the `new_` is implied